accord-core/src/main/java/accord/topology/Shard.java

```diff
@@ -41,27 +41,34 @@ public class Shard
     public final List<Id> nodes;
     public final ExtendedSortedArrayList<Id> sortedNodes;
     public final Set<Id> fastPathElectorate;
-    public final Set<Id> joining;
+    public final Set<Id> joining, leaving;
```

Comment:
in accord we don't leverage this, though it would be nice... we have cases where we ask nodes that are leaving for information, and then rely on timeouts to select from another peer.  I didn't want to change this behavior in this patch, but `leaving` is important during C* boot up so it knows which nodes to mark removed